### DISCUSSION TITLE
Tournament Client no longer allow user to set resolution lower than the resolution the monitor running

### DISCUSSION THREAD

**Discussion by xuanhaivptthn** [ppy/osu#24670] [Q&A] [answered]
As of build 2023.815.0-lazer, Tournament Client no longer allow us to do so
For example, I have a 1366x768 monitor, and I cannot set client's height to maybe 564 anymore. Can't even resize the windows manually

![image](https://github.com/ppy/osu/assets/46238512/8dba46a7-b0c9-4661-ba4e-40360858d7c8)
(Screenshot looks weird because I had to use my phone as an external monitor temporarily)

Tried Fullscreen, Borderless and Windowed with no better result
Is this an intended feature/expected limitation, or just a bug that will be fixed later on?

**Comment by Kuina-chan** [comment]
Can still resize but only making it bigger.

**Comment by xuanhaivptthn** [comment]
Seems like intentional  https://github.com/ppy/osu/pull/24348
I guess this only affect a small number of user, but still worth a fix
![image](https://github.com/ppy/osu/assets/46238512/ef34611a-3bee-4575-a696-0df03b8b62bf)
Only the left bar is not even close to usable for editting

**Comment by peppy** [comment]
The minimum height is 768 (in line with minimum width of 1366). What's the scenario where you want to make it less tall? 720p streaming?

**Comment by xuanhaivptthn** [comment]
Either streaming 720p or updating bracket.json

**Comment by peppy** [comment, marked as answer]
Will be fixed via https://github.com/ppy/osu/pull/24680
